thrimbletrimmer/edit.html

```diff
@@ -196,6 +196,7 @@ <h2>Keyboard Shortcuts</h2>
 			<div id="submission-toolbar">
 				<button id="submit-button">Submit</button>
 				<button id="save-button">Save Draft</button>
+				<button id="submit-changes-button">Submit Changes</button>
```

Comment:
Is this meant to start hidden if the video isn't done yet?

Reply:
ah, yes it is. So I'd want `<button id="submit-changes-button" class="hidden">` ?

Reply:
Yup, that'll start it hidden (and is exactly how I'd make it hidden on page load).